index.html

```diff
@@ -413,13 +413,24 @@ <h2>JOSE Header Parameters and JWT Claims</h2>
         defined in <a data-cite="VC-DATA-MODEL-2.0#validity-period">Validity Period</a>,
         which represent the validity of the data that is being secured.
       </p>
+      <p>
+        The claims and security provided by this specification are independent of the data 
+        secured and semantics provided by the [[VC-DATA-MODEL-2.0]]. This means that while the security
+        features of this specification ensure data integrity and authenticity, they do
+        not dictate the interpretation of claim data.
+      </p>
+      <p>
+        Implementers SHOULD avoid setting JWT claims to values that conflict with
+        verifiable credential properties, especially with pairs such as
+        `iss` and `issuer`, `jti` and `id`, and `sub` and `credentialSubject.id`.
```

Comment:
@David-Chadwick — Answering [your comment](https://github.com/w3c/vc-jose-cose/pull/226#issuecomment-1944514573) --
> @decentralgabe This PR was merged without addressing my requested editorial change.

I *think* you're referring to [this comment](https://github.com/w3c/vc-jose-cose/pull/226#discussion_r1483325600) as your "requested editorial change".

Please note that your intent was not obvious because you did not submit it as a change request using the GitHub tooling, which would have shown the existing PR content in contrast to your suggestion, as you will see below, in my suggested massage of your suggestion:

```suggestion
        Implementers SHOULD avoid setting JWT claims to values that conflict with
        the values of verifiable credential properties when a claim and property 
        pair refer to the same conceptual entity, especially with pairs such as
        `iss` and `issuer`, `jti` and `id`, and `sub` and `credentialSubject.id`.
        For example, JWK claim `iss` should not be set to a value which conflicts
        with the value of verifiable credential property `issuer`.
```

I think a new PR (possibly starting with a new issue) will be needed to enact this (to my eyes, editorial) change, which I believe should only touch this paragraph.

Reply:
Correct. It is only this paragraph that needs changing.  But note that the GitHub tooling did not work (again) otherwise I would have used it.